src/components/ChatLog.js

```diff
@@ -0,0 +1,27 @@
+import { React } from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatMessageArray = props.chatsMessages.map((chat) => {
+    return (
+      <ChatEntry
+        sender={chat.sender}
+        body={chat.body}
+        timeStamp={chat.timeStamp}
+        id={chat.id}
+        key={chat.id}
+        liked={chat.liked}
+        onChangeLike={props.onChangeLike}
+      />
+    );
+  });
+  return chatMessageArray;
+};
+
+ChatLog.propTypes = {
+  chatsMessages: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Since we know the shape of the object that is in the array, we should explicitly validate the shape of the object here for an added layer of validation to prevent your protect your program from errors. 

Also note that you have .isRequired on arrayOf() but you'll also. need to have .isRequired on the elements inside arrayOf()

```javascript
ChatLog.propTypes = {
  chatsMessages: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.number.isRequired,
    sender: PropTypes.string.isRequired,
    body: PropTypes.string.isRequired,
    timeStamp: PropTypes.string.isRequired,
    liked: PropTypes.bool
  })).isRequired,
  onChangeLike: PropTypes.func.isRequired
};
```